### PR TITLE
Create README_SWA.md

### PR DESCRIPTION
**Pull request by mkaomwakuni**
Added a Swahili Language Translation

## Guidelines
The Idea behind adding the Swahili Language Translation is to be able to allow understanding  of the android roadmap especially for the developers from Tanzania and several east African regions.

### Types of changes
What types of changes does your code introduce?

- [ ] Language (Swahili Translation)

### Preparing a pull request for review
Ensure your change is properly formatted by running:

```gradle
$ ./gradlew spotlessApply
```

Please correct any failures before requesting a review.

### CODE REVIEW

File: README_SWA.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+</p>
+
+
+Ramani ya Barabara ya Wasanidi Programu wa Android 2022 inapendekeza njia za kujifunza ili kuelewa usanidi wa Android. Unaweza kusoma ramani ya barabara kufuatia njia ya mstari katikati ya ramani. <br>
+
+Kila nodi huonyesha dhana za mifumo ya Android, Android SDK, na maktaba zinazotumika kwa ujumla. Ingefaa kurejelea [Marejeleo ya Wasanidi Programu wa Android](https://developer.android.com/reference) au GitHub kwa istilahi mahususi. <br>
+
+Kwa kuongeza, **huhitaji kujifunza kila kitu kutoka kwa ramani hii ya barabara**. Kwa hivyo, tunapendekeza usome tu sehemu ambazo zitakusaidia.
+## 🗺 Ramani ya barabara
```
Comment (skydoves) [inline README_SWA.md:20]: We can a new line here.

**Review by skydoves — CHANGES_REQUESTED** [review]
Hi @mkaomwakuni, thank you for your contribution!
Could you update other README files like the other PRs? You can reference: https://github.com/skydoves/android-developer-roadmap/pull/41

**Review by mkaomwakuni — COMMENTED** [review]
good

**Review by skydoves — APPROVED** [review]
Looks good to me!

**Pull request merged**